scripts/zkai_d128_two_slice_recursive_pcd_backend_gate.py

```
@@ -0,0 +1,1048 @@
+            raise D128TwoSliceRecursivePCDBackendError(f"mutation case {index} was accepted")
+        require_str(item["rejection_layer"], f"case {index} rejection_layer")
+        require_str(item["error"], f"case {index} error")
+
+
+def validate_payload(payload: dict[str, Any]) -> None:
+    validate_core_payload(payload)
+    expected_core = build_core_payload()
+    expected_cases = run_mutations(expected_core)
+    validate_cases(payload["cases"], expected_cases)
+    expect_equal(payload["case_count"], len(expected_cases), "case_count")
+    expect_equal(payload["all_mutations_rejected"], True, "all_mutations_rejected")
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)